docs/experimental-features.md

```diff
@@ -427,7 +427,14 @@ ipfs config --json Experimental.ShardingEnabled true
 
 ### In Version
 
-0.4.14
+0.4.14 :
+  - Introduced
+
+0.5.0 : 
+   - No longer needs to use the DHT for the first resolution
+   - When discovering PubSub peers via the DHT, the DHT key is different from previous versions
+      - This leads to 0.5 IPNS pubsub peers and 0.4 IPNS pubsub peers not being able to find each other in the DHT
+   - Robustness improvements
```

Comment:
@Stebalien is this the type of version history you were looking for?

Reply:
lgtm